code/game/turfs/simulated/wall/mineral_walls.dm

```diff
@@ -303,3 +303,19 @@
 /turf/closed/wall/mineral/plastitanium/copyTurf(turf/T)
 	. = ..()
 	T.transform = transform
+
+/////////////////////Pirate Plastitanium walls/////////////////////
+
+/turf/closed/wall/mineral/plastitanium/pirate
+	desc = "Yarr just try to blow this to smithereens!"
+	explosion_block = 50
+	canSmoothWith = list(/turf/closed/wall/mineral/plastitanium/pirate, /obj/machinery/door/airlock/shuttle, /obj/machinery/door/airlock, /obj/structure/window/plastitanium/pirate, /obj/structure/shuttle/engine, /obj/structure/falsewall/plastitanium)
+
+/turf/closed/wall/mineral/plastitanium/nodiagonal/pirate/
+	explosion_block = 50
+
+/turf/closed/wall/mineral/plastitanium/nosmooth/pirate/
+	explosion_block = 50
+
+/turf/closed/wall/mineral/plastitanium/overspace/pirate/
```

Comment:
Also you'll fuck up the smoothing if you don't move the `/pirate` path before `/overspace`, `/nosmooth` and `/nodiagonal`. 
Refer to https://github.com/Citadel-Station-13/Citadel-Station-13/blob/2acb52fb56a7ba8e3706f5e08bd515c3620f3588/code/game/turfs/simulated/wall/mineral_walls.dm#L261 for types' values.